.github/workflows/e3sm-gh-pages.yml

```diff
@@ -15,7 +15,7 @@ concurrency:
   
 jobs:
   Build-and-Deploy-docs:
-    if: ${{ github.event.repository.name != 'scream' }}
+    if: ${{ github.event.repository.name == 'e3sm' }}
```

Comment:
Better compare to `e3sm`, or else this would run on any non-scream fork of e3sm...

Reply:
Ah, good point. Let me check

Reply:
I think they are not, as stated [here](https://docs.github.com/en/actions/learn-github-actions/expressions):

```
GitHub ignores case when comparing strings.
```